frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -32,14 +32,14 @@ pods:
             port: {{TASKCFG_ALL_CASSANDRA_STORAGE_PORT}}
           ssl:
             port: {{TASKCFG_ALL_CASSANDRA_SSL_STORAGE_PORT}}
-          native-client:
+          node:
             port: {{TASKCFG_ALL_CASSANDRA_NATIVE_TRANSPORT_PORT}}
             advertise: true
-          {{#TASKCFG_ALL_CASSANDRA_START_RPC}}
-          thrift-client:
+            vip:
+              prefix: node
+              port: 9042
+          rpc:
```

Comment:
this endpoint will only be available if START_RPC is enabled. its arguably a bug to display it when the feature is disabled

Reply:
Yep. @nickbp can you create a ticket to make just this fix in 0.30.X?